configs/slurm_local.yml

```diff
@@ -0,0 +1,13 @@
+{
+  "data-path": "data/enron/enron_text_document",
+  "vocab-file": "data/gpt2-vocab.json",
+  "merge-file": "data/gpt2-merges.txt",
+  "save": "checkpoints",
```

Comment:
Add a comment explaining why this is different from `configs/local_setup.yml` (i.e. why you capitalized bools)

Reply:
Oops that's not possible. Comments aren't parsed correctly by srun. We should document this, but elsewhere.